packages/devtools/cli-next/src/commands/chat/components/App.tsx

```diff
@@ -126,14 +135,40 @@ export const App = (props: AppProps) => {
     // Replay logs once.
     props.logBuffer?.replay();
     log.info('focus console then ctrl-s to save logs to file');
+
+    // Repaint in case terminal is cleared (via cmd-k -- which is not propagated to the app from the terminal).
```

Comment:
Is this a workaround? It seems like a very slow interval for repainting based on an action, todo to improve it?